scripts/plot_network.py

```diff
@@ -261,7 +261,7 @@ def split_costs(n):
     map_figsize = opts['map']['figsize']
     map_boundaries = opts['map']['boundaries']
 
-    n = load_network(snakemake.input.network, snakemake.input.tech_costs, snakemake.config)
+    n = load_network_for_plot(snakemake.input.network, snakemake.input.tech_costs, snakemake.config)
```

Comment:
This should probably be ...plots?
```suggestion
    n = load_network_for_plots(snakemake.input.network, snakemake.input.tech_costs, snakemake.config)
```